src/components/report-technical-issue/template.test.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env jest */
+
+const axe = require('../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('report-technical-issue')
+
+describe('Report Technical Issue', () => {
+  describe('by default', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('report-technical-issue', examples.default)
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('renders a link element', () => {
+      const $ = render('report-technical-issue', examples.default)
+
+      const $component = $('.govuk-link')
+      expect($component.get(0).tagName).toEqual('a')
+      expect($component.attr('target')).toEqual('_blank')
+      expect($component.attr('rel')).toEqual(undefined)
```

Comment:
I was looking at this in relation to the `new-tab-link`, that specifically runs with `noopener noreferrer` which I feel are actively inappropriate for this usage.  I wanted to document that it shouldn't have those.  If we agree that it shouldn't have a `rel` it might be worth putting that in its own test and naming it more clearly.